Orange/canvas/__main__.py

```diff
@@ -180,32 +180,55 @@ def make_sql_logger(level=logging.INFO):
     sql_log.addHandler(handler)
 
 
-def show_survey():
-    # If run for the first time, open a browser tab with a survey
-    settings = QSettings()
-    show_survey = settings.value("startup/show-survey", True, type=bool)
+def setup_notifications():
+    settings = config.settings()
+
+    # If run for the fifth time, prompt short survey
+    show_survey = settings["startup/show-short-survey"] and \
+                  settings["startup/launch-count"] >= 5
     if show_survey:
-        question = QMessageBox(
-            QMessageBox.Question,
-            'Orange Survey',
-            'We would like to know more about how our software is used.\n\n'
-            'Would you care to fill our short 1-minute survey?',
-            QMessageBox.Yes | QMessageBox.No)
-        question.setDefaultButton(QMessageBox.Yes)
-        later = question.addButton('Ask again later', QMessageBox.NoRole)
-        question.setEscapeButton(later)
-
-        def handle_response(result):
-            if result == QMessageBox.Yes:
+        surveyDialogButtons = NotificationWidget.Ok | NotificationWidget.Close
+        surveyDialog = NotificationWidget(icon=QIcon("Orange/widgets/icons/information.png"),
```

Comment:
These icon paths must be absolute. They will not be located unless running at top level installation directory.

Reply:
I used `Orange.widgets.gui.resource_filename()` in place of absolute paths. Consider this temporary until we receive custom made icons, which will be placed in `Orange/canvas/icons`, retrieved with `canvas_icons()`.